### PR TITLE
[semver:patch] Fix issue when `jq` is not on the executor

### PR DESCRIPTION
**Pull request by PPFerrari**
ref https://github.com/ClouDesire/support/issues/5288

### CODE REVIEW

File: src/scripts/git_checkout.sh [hunk 1]
```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 
 CheckoutRepo() {
+
+  if ! command -v 'jq' &> /dev/null; then
+    wget 'https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64' -O '/usr/local/bin/jq'
```
Comment (gionn) [inline src/scripts/git_checkout.sh:6]: check for wget or curl command too, or fails

File: src/scripts/github_pr_read_labels.sh [hunk 2]
```diff
@@ -6,6 +6,12 @@ ReadPRLabels() {
     exit 0
   fi
 
+  if ! command -v 'jq' &> /dev/null; then
+    wget 'https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64' -O '/usr/local/bin/jq'
+    chmod +x '/usr/local/bin/jq'
+    jq --version
+  fi
+
```
Comment (gionn) [inline src/scripts/github_pr_read_labels.sh:14]: avoid duplication?

**Review by gionn — CHANGES_REQUESTED** [review]

File: src/commands/git_merge_default.yml [hunk 3]
```diff
@@ -5,6 +5,9 @@ steps:
   - run:
       name: Setup git
       command: <<include(scripts/git_setup.sh)>>
+  - run:
+      name: "Setup prerequisites"
+      command: <<include(scripts/setup.sh)>>
```
Comment (PPFerrari) [inline src/commands/git_merge_default.yml:10]: dentro `setup.sh` vogliamo introdurre anche il contenuto di `git_setup.sh`?

File: src/scripts/setup.sh [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  jq_installation_path='/usr/local/bin/jq'
+
+  if ! command -v 'jq' &> /dev/null; then
+    if ! command -v 'wget' &> /dev/null; then
+      echo "ERROR: impossible to find command wget. Trying with curl..."
+      if ! command -v 'curl' &> /dev/null; then
+        echo "ERROR: impossibile to find command curl. Exiting..."
+        exit 1
+      else
+        DOWNLOAD_CMD="curl ${jq_url} "
```
Comment (gionn) [inline src/scripts/setup.sh:18]: curl -o path

File: src/scripts/setup.sh [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    JQ_VERSION="1.6"
+  fi
+
+  jq_url="https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64"
+  jq_installation_path='/usr/local/bin/jq'
+
+  if ! command -v 'jq' &> /dev/null; then
+    if ! command -v 'wget' &> /dev/null; then
+      echo "ERROR: impossible to find command wget. Trying with curl..."
+      if ! command -v 'curl' &> /dev/null; then
```
Comment (gionn) [inline src/scripts/setup.sh:14]: you can avoid nested ifs by checking if DOWNLOAD_CMD is empty before the end

**Review by gionn — CHANGES_REQUESTED** [review]

**Review by gionn — APPROVED** [review]

**Pull request merged**